### DISCUSSION TITLE
Advantages of using nuxt-start standalone over nuxt start

### DISCUSSION THREAD

**Discussion by gd000** [nuxt/nuxt#9437] [Questions (Nuxt 2)] [answered]
Hello, 

I noticed there is a [nuxt-start](https://www.npmjs.com/package/nuxt-start) package on npm, linking to this github page, and I would like to know if there are advantages of using it instead of nuxt's "nuxt start", when deployed in production mode.

I saw it is ~3 times smaller than nuxt, so it will decrease the size of the project on my server.
Its dependencies are also different : less @nuxt dependencies, but demands some vue packages...

Thanks in advance.

**Comment by danielroe** [comment, marked as answer]
@gd000 `nuxt-start` source can be found [here](https://github.com/nuxt/nuxt.js/tree/dev/distributions/nuxt-start). It is intended to be a lightweight way to start a built Nuxt server, without all the dependencies that are only needed in development and at build-time.

**Reply by gd000** [reply]
Thank you !
